CHANGELOG.md

```diff
@@ -6,8 +6,13 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## [Unreleased] - ReleaseDate
 ### Added
 
+- Documentation of the `sys::wait` module
```

Comment:
You don't need to put this in the CHANGELOG, because it doesn't change the user experience in any way.

Reply:
:+1: